h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -1915,13 +1915,13 @@ protected PredictScoreResult predictScoreImpl(Frame fr, Frame adaptFrm, String d
 
   @Override
   public boolean haveMojo() {
-    if (_parms.interactionSpec() == null) return super.haveMojo();
+    if (!_parms._HGLM && _parms.interactionSpec() == null) return super.haveMojo();
     else return false;
   }
 
   @Override
   public boolean havePojo() {
-    if (_parms.interactionSpec() == null) return super.havePojo();
+    if (!_parms._HGLM && _parms.interactionSpec() == null) return super.havePojo();
```

Comment:
@wendycwong please confirm, I reverted the changes in GLM driver (haveMojo/Pojo always return true).